void/tests/test_math.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+
+import numpy as np
+
+from void import math_utils
+
+
+class CalculatePolyTest(unittest.TestCase):
+    @staticmethod
+    def _round(values):
+        return [list(map(lambda x: round(x, 4), x)) for x in values]
+
+    def test_calculate_45(self):
+        values = math_utils.calculate_poly((0, 0), 3, 3, 315)
+        values = self._round(values)
+        expected = [[0, -2.1213], [-2.1213, 0], [0, 2.1213], [2.1213, 0]]
+        expected = np.asarray(expected)
+        expected = math_utils.sort_ndarray(expected)
+        try:
+            np.testing.assert_almost_equal(values, expected, decimal=4)
+            res = True
+        except AssertionError as err:
```

Comment:
Way to go, `np.testing` is nice 👌!
But why are we catching `AssertationError` only to raise it from a different function? Can we replace the whole `try...except...` with just `np.testing.assert_almost...`?

Reply:
Done.